package.json

```diff
@@ -50,6 +50,7 @@
   },
   "dependencies": {
     "@jupyter-widgets/base": "^1.1.10 || ^2 || ^3 || ^4 || ^5 || ^6",
+    "@jupyterlab/galata": "^5.2.2",
```

Comment:
```suggestion
```

I believe we don't need this

Reply:
We probably need it because when I committed it, it broked the CI test.